app/views/posts/show.html.erb

```diff
@@ -1 +1,52 @@
-<h1>this is going to show the user post page</h1>
\ No newline at end of file
+<section class="my-bio bg-secondary bg-gradient">
```

Comment:
- [ ] On this page show view, kindly add a button for creating a like and ensure you implement the controller part of this feature